OrbitVulkanLayerClientGgp/include/OrbitVulkanLayerClientGgp/DispatchTable.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_CLIENT_GGP_DISPATCH_TABLE_H_
+#define ORBIT_VULKAN_LAYER_CLIENT_GGP_DISPATCH_TABLE_H_
+
+#include <vulkan/vulkan_core.h>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/synchronization/mutex.h"
+#include "vulkan/vk_layer_dispatch_table.h"
+#include "vulkan/vulkan.h"
+
+class DispatchTable {
```

Comment:
Please add a brief comment describing the purpose and the usage of this class.

Reply:
Done, thanks!